moveit_demo_nodes/run_moveit_cpp/src/run_moveit_cpp.cpp

```diff
@@ -94,7 +94,9 @@ class MoveItCppDemo
 
     // Set joint state goal
     RCLCPP_INFO(LOGGER, "Set goal");
-    arm.setGoal("home");
+    auto robot_state = *moveit_cpp_->getCurrentState();
```

Comment:
Since we removed the `home` pose, I used a robot state to set the goal

Reply:
I switched it to the extended pose